app-shells/fish/patches/fish-3.7.1.patchset

```diff
@@ -0,0 +1,34 @@
+From bb8f751b7ef40a63f7b2eb2f6df9d5c257532f92 Mon Sep 17 00:00:00 2001
+From: Ken mays <kmays2000@gmail.com>
+Date: Tue, 22 Oct 2024 05:12:03 +0000
+Subject: Fix std::wcsto* issue for Haiku
```

Comment:
What's the issue actually? Is it reported in Haiku?

Reply:
https://dev.haiku-os.org/ticket/18207

Reply:
Duriing fish's 'configuration', spotted 'Looking for wcstod_l - not found'.... GCC13/Haiku x86.

Reply:
Please add the ticket URL in the patch message.